src/qt/optionsmodel.cpp

```diff
@@ -126,6 +126,8 @@ void OptionsModel::Init(bool resetSettings)
         settings.setValue("fUseProxy", false);
     if (!settings.contains("addrProxy"))
         settings.setValue("addrProxy", "127.0.0.1:9050");
+    if (!settings.value("addrProxy").toString().contains(':'))
```

Comment:
Can you just make this an OR with the if statement above?

Reply:
Done